astroquery/esasky/core.py

```diff
@@ -585,15 +586,19 @@ def _remove_extra_herschel_directory(self, file_and_directory_name,
                                          directory_path):
         full_directory_path = os.path.abspath(directory_path)
         file_name = file_and_directory_name[file_and_directory_name.index("/") + 1:]
+        
+        print("full path: %s" %(full_directory_path))
```

Comment:
could you use the logger for any prints (e.g these seems to be ``log.info``), or introduce a verbose variable?